q2_longitudinal/_longitudinal.py

```diff
@@ -207,15 +217,10 @@ def volatility(output_dir: str, metadata: qiime2.Metadata,
         raise ValueError('state_column must contain at least two unique '
                          'values.')
 
-    categorical = metadata.filter_columns(column_type='categorical')
-    numeric = metadata.filter_columns(column_type='numeric')
-
-    # Ensure the default_* columns are members of their respective groups.
-    # This will raise a uniform framework error on our behalf if necessary.
-    categorical.get_column(default_group_column)
-    numeric.get_column(default_metric)
-
-    group_columns = list(categorical.columns.keys())
+    # We want to include the original Sample ID in the viz, so reset the index.
+    data = metadata.to_dataframe().reset_index(drop=False)
```

Comment:
Ah, thanks for the clarification - I completely misunderstood #83. Should I keep this change, in addition to the `individual_id_column`?

Reply:
My fault, #83 does not have enough description. The goal is to calculate mean lines across all samples, rather than splitting by group. Your change does 50% of that — there is no group, but there is no mean line, either. So we need a way to group all samples together for calculating a "group" mean, perhaps by generating a new metadata column (though that risks overwriting an existing column that happens to be named something like "NA").

For some reason I thought that this change was actually targeting #90 — with the `individual_id_column` change added in you can claim that issue too! or not, if you just want to get this off your plate — that was not in the original manifest for this PR so I would understand.